dotnetapp-buildcontainer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet:core
```

Comment:
Sure, I'll change it. IIRC I thought I still had a lot of work to do on this one so I hadn't done a pass looking for this sort of stuff.